cra-tf-validate-ignore-rules.json

```diff
@@ -1,3 +1,22 @@
 {
-    "scc_rules": []
+    "scc_rules": [
+        {
+            "scc_rule_id": "rule-4d86c074-097e-4ff3-a763-ccff128388e2",
+            "description": "Check whether multifactor authentication (MFA) is enabled at the account level",
+            "ignore_reason": "In order for this rule to pass, multifactor authentication (MFA) is enabled at the account level (tracking in https://github.ibm.com/workload-eng-services/HPCCluster/issues/3422).",
```

Comment:
This rule is only happening because we added an account level data lookup. Since we do not create any account setting in this DA, it can be ignored. That should be the description here. Remove the link to the HPC issue - thats not even our teams issues

Reply:
this is still not resolved